Shakefiles/Extra.hs

```diff
@@ -15,3 +16,12 @@ phonyPrefix prefix action =
 forEach :: Monad m => [a] -> (a -> m ()) -> m ()
 forEach list f =
     mapM_ f list
+
+
+--ghc-option required because of https://gitlab.haskell.org/ghc/ghc/-/issues/20592
```

Comment:
💯 Thanks for the issue link!  And hopefully this will be fixed in ghc 9.4 🙏

Reply:
FYI, this issue is supposedly fixed in ghc 9.2.3.  (No need to update this PR yet -- I'm doing a bit of other maintenance first.)

Reply:
This is apparently still a problem with ghc 9.2.5.

But I ended up reverting the changes here to apply this in the Shakefiles, and instead it needs to be set in the outside environment.  (I'm not sure that's the ideal choice, but I'm gonna stick with it since it's working now for arm64 CI).  Here's the change I settled on: https://github.com/avh4/elm-format/pull/784/commits/4e5113a6a7014680302866a57da776afe0598e9d